var/spack/repos/builtin/packages/freesasa/package.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Freesasa(AutotoolsPackage):
+    """C-library for calculating Solvent Accessible Surface Areas"""
+
+    homepage = "http://freesasa.github.io"
+    url = "https://github.com/mittinatten/freesasa/archive/refs/tags/2.1.2.tar.gz"
```

Comment:
Can we use [this artifact](https://github.com/mittinatten/freesasa/releases/download/2.1.2/freesasa-2.1.2.zip) to have a stable shasum to check?

Reply:
Is there an advantage in using the artefact instead of the `Source code (tar.gz)`? I've found instances (in other projects) where such artifacts were missing, while `Source code (tar.gz)` is always there by default.

Reply:
Yes, that should give more stability to the sha256sum we use. The `.tar.gz` is generated on demand, so any upstream change from GitHub may impact the hash.  See https://github.com/orgs/community/discussions/46034

Reply:
Obviously, if there is no artifact, using the `tar.gz` is fine, but if there's a viable artifact, that should be preferred.

Reply:
```suggestion
    url = "https://github.com/mittinatten/freesasa/releases/download/2.1.2/freesasa-2.1.2.zip"
```


I was not aware of this issue, thank you for pointing it out.